data-model/SDP-sample-records/codicological-unit_sample1.json

```diff
@@ -0,0 +1,143 @@
+      },
+      "translation": {
+        "en": ["May there be a blessed rememberance before God and his Christ for Jacob and Bar-Shabbon, sons of Basil, from the village of Yw."]
+      },
+      "note": ["The association between these persons and the scribe, or the production of the manuscript more generally, remains unclear."],
+      "features": []
+    }
+  ],
+  "associated_dates": [
+    {
+      "type": "production",
+      "date_human_readable": "6th c. CE",
+      "date_normalized": "0500/0600",
+      "source": "Kamil 1970, based on paleographic data."
+    }
+  ],
+  "associated_persons": [{
+      "type": "unknown",
+      "id": "https://sinaimanuscripts/library.ucla.edu/person/1",
+      "attested_name_NS": "ܝܥܩܘܒ ܒܪܐ ܕܒܣܝܠ",
+      "attested_name": "Ya'qub bar d-Basil",
+      "source": "Attested in the note on f. 114v"
+    },
+    {
+      "type": "unknown",
+      "id": "https://sinaimanuscripts/library.ucla.edu/person/2",
+      "attested_name_NS": "ܒܪ ܫܒܘܢ ܒܪܐ ܕܒܣܝܠ",
+      "attested_name": "Bar-Shabon bar d-Basil",
+      "source": "Attested in the note on f. 114v"
+    }
+  ],
+  "associated_places": [{
+      "type": "production",
+      "id": "https://sinaimanuscripts/library.ucla.edu/place/1",
+      "attested_name_NS": "ܛܘܪ ܣܝܢܝ",
+      "attested_name": "Mount Sinai",
+      "source": "Attested in the colophon on f. 152v"
+    },
+    {
+      "type": "unknown",
+      "id": "https://sinaimanuscripts/library.ucla.edu/place/2",
+      "attested_name_NS": "ܐܝܘ ܩܪܝܬܐ",
+      "attested_name": "The Village Yw",
+      "source": "Provided in connection with Jacob and Bar-Shabbon in the note on f. 114v"
+    }
+  ],
+  "provenance": [],
+  "decorations": [{
+    "note": ["Geometric decorations surround the rubrics (e.g., f. 114r)."],
+    "features": ["Decoration, Geometric"],
+    "extent": ["f. 114r"]
+  }],
+  "features": ["Rubrication", "Headpiece(s)"],
+  "references": [{
+      "uri": "https://www.zotero.org/groups/2276183/items/EUCERC5C",
+      "citation_key": "Lewis 1894",
+      "cited_range": [{
+        "unit": "p.",
+        "range": "1-2"
+      }]
+    },
+    {
+      "uri": "https://www.zotero.org/groups/2276183/items/9IEZ55UL",
+      "citation_key": "Kamil 1970",
+      "cited_range": [{
+        "unit": "p.",
+        "range": "152"
+      }, {
+        "unit": "number",
+        "range": "38"
+      }]
+    },
+    {
+      "uri": "https://www.zotero.org/groups/2276183/items/MPHJNWZL",
+      "citation_key": "Géhin 2017",
+      "cited_range": [{
+        "unit": "p.",
+        "range": "28-30"
+      }]
+    }
+  ],
+  "bibliography": [],
+  "iiif": {
+    "iiif_manifest": "https://iiif.library.ucla.edu/ark%3A%2F21198%2Fz1x07bdf/manifest",
+    "text_direction": "right-to-left",
+    "viewing_hint": "paged",
+    "range": ""
+  },
+  "contributor": ["gkessel"],
+  "parent_objects": ["ark:/21198/z1x07bdf"],
+  "textual_artifacts": [{
+    "id": "ark:/21198/z1x07bdf",
+    "sequence": 1,
+    "layer": 0
```

Comment:
remove layer for now. Add to confluence page about incorporating palimpsest into the data model.